modules/apps/staging/staging-test/src/testIntegration/java/com/liferay/staging/test/StagingImplTest.java

```diff
@@ -215,6 +217,51 @@ public void testGetRemoteLayout() throws Exception {
 			remoteLiveGroupLayout.getTitle());
 	}
 
+	@Test
+	public void testGetRemoteLayoutPlid() throws Exception {
+		enableRemoteStaging(false);
+
+		Layout remoteStagingGroupLayout = LayoutTestUtil.addTypePortletLayout(
+			_remoteStagingGroup);
+
+		try {
+			long remoteLiveGroupLayoutPlid = _executeWithRemoteCredentials(
+				() -> StagingUtil.getRemoteLayoutPlid(
+					TestPropsValues.getUserId(),
+					remoteStagingGroupLayout.getGroupId(),
+					remoteStagingGroupLayout.getPlid()));
+
+			Assert.assertEquals(0L, remoteLiveGroupLayoutPlid);
+
+			Map<String, String[]> parameters =
+				ExportImportConfigurationParameterMapFactoryUtil.
+					buildFullPublishParameterMap();
+
+			StagingUtil.publishLayouts(
+				TestPropsValues.getUserId(), _remoteStagingGroup.getGroupId(),
+				_remoteLiveGroup.getGroupId(), false, parameters);
+
+			remoteLiveGroupLayoutPlid = _executeWithRemoteCredentials(
+				() -> StagingUtil.getRemoteLayoutPlid(
+					TestPropsValues.getUserId(),
+					remoteStagingGroupLayout.getGroupId(),
+					remoteStagingGroupLayout.getPlid()));
+
+			Layout remoteLiveGroupLayout = LayoutServiceUtil.fetchLayout(
+				_remoteLiveGroup.getGroupId(),
+				remoteStagingGroupLayout.isPrivateLayout(),
+				remoteStagingGroupLayout.getLayoutId());
+
+			Assert.assertNotNull(remoteLiveGroupLayout);
+
+			Assert.assertEquals(
+				remoteLiveGroupLayout.getPlid(), remoteLiveGroupLayoutPlid);
+		}
+		catch (NoSuchLayoutException noSuchLayoutException) {
```

Comment:
hey @DaniSzimko I think this is redundant. We can remove the `try` block and just let the exception bubble up, which will cause the test to fail anyway, and with a much clearer error message.